ui/src/components/Code/CodeBlock.tsx

```diff
@@ -22,6 +25,12 @@ interface CodeBlockProps {
 
 export default function CodeBlock({ header, tabs }: CodeBlockProps) {
   const [activeTab, setActiveTab] = useState(0);
+  const editorRef = useRef<MonacoEditorType>(null);
+
+  const [originalContentHeight, setOriginalContentHeight] = useState(0);
+  const [disableWordWrap, setDisableWordWrap] = useState(false);
+  const [wordWrap, setWordWrap] = useState('off');
```

Comment:
Are there any values this can be besides `'on'` and `'off'`? Wondering if it should be a boolean